src/pages/en/docs/vision.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Vision
+description: Email Markup Consortium's vision for more consistent rendering in email clients.
+layout: "@layouts/MainLayout"
+---
+
+## Background
+
+Email clients have the challenge of dealing with third-party code: HTML email messages. For an email client to safely embed a HTML message, they need to:
+
+1. Sanitize the HTML message
+2. Embed the HTML message
+
+### Third-party content sanitization
+
+Third-party content sanitization is a problem any app/website must address if they wish to safely process or display third-party content. That is, only what is considered to be safe is kept and anything else is either removed or replaced.
```

Comment:
Is this second sentence strictly accurate? I was under the impression that some operate allowlists while others use blocklists. Minor distinction but maybe worth rewording if my understanding is correct.

Reply:
This paragraph describes third-party content sanitization in general; not just in the context of email. It 

I think this can be expanded to describe the allowlist/blocklist approaches that different email clients may take. The following paragraph starts with "In the context of email" so it may make sense to include this there.

Reply:
Makes sense. Not a blocker by any means.

Reply:
I'll merge this as is for now, but I'm happy to update this. I'll open an issue.